controllers/common/common_status_update.go

```diff
@@ -44,14 +44,26 @@ func RootStatusUpdate(ctx context.Context, c client.Client, rootPolicy *policies
 
 	complianceState := CalculateRootCompliance(cpcs)
 
+	log.Info("Sorting the root policy status", "RootPolicyName", rootPolicy.Name, "Namespace", rootPolicy.Namespace)
+	sort.SliceStable(placements, func(i, j int) bool {
+		pi := placements[i].PlacementBinding + " " + placements[i].Placement + " " +
+			placements[i].PlacementRule + " " + placements[i].PolicySet
+		pj := placements[j].PlacementBinding + " " + placements[j].Placement + " " +
+			placements[j].PlacementRule + " " + placements[j].PolicySet
+
+		return pi < pj
+	})
+	sort.SliceStable(cpcs, func(i, j int) bool {
+		return cpcs[i].ClusterName < cpcs[j].ClusterName
+	})
```

Comment:
This list is already sorted in `CalculatePerClusterStatus`. If that needs to be updated to be a stable sort, I think it should be updated there.

Maybe sorting the `placements` should also be done in its function... but it doesn't make a big difference.

Reply:
maybe I can remove this. `placements` which function you mean?

Reply:
`GetClusterDecisions` ?